packages/playground/src/views/nodes.vue

```diff
@@ -466,6 +477,9 @@ export default {
       isDialogOpened.value = true;
     };
 
+    function reloadTable() {
+      setTimeout(loadNodes, 20000);
+    }
```

Comment:
updating nodes table after 20s is neither good ux nor fix to waiting because node might be not update within that 20s
Also we do we need to reloadData we can just update that row

what do u think? @AhmedHanafy725

Reply:
sleeping before reloading the data is required as graphql processor takes some time to get the update.

Reply:
@MohamedElmdary fixed